### PR TITLE
Add Booleans Assignment

### PR DESCRIPTION
**Pull request by eddyannobil**
Add the Booleans Homework to the repository

### CODE REVIEW

**Comment by eddyannobil** [comment]
> Add the Booleans Homework to the repository

@fiifikrampah Have added this task as requested

**Comment by fiifikrampah** [comment]
@eddyannobil This is the third assignment right? So I think we should merge this after the 2nd one gets merged.

**Review by fiifikrampah — APPROVED** [review]
Looks good!

**Pull request merged**
